src/hooks/vaultsHooks.ts

```diff
@@ -304,6 +304,40 @@ export const useVaultFormSlippage = () => {
   return mapNullableQueryResult(slippageQueryResult);
 };
 
+export const useVaultCalculationForLaunchingMarket = ({ amount }: { amount: number }) => {
+  const accountInfo = useAppSelector(selectSubaccountStateForVaults);
+  const slippageResponse = useVaultFormSlippage().data;
+  const vaultAccount = useLoadedVaultAccount().data;
+
+  const vaultFormInfo = useMemo(
+    () => new VaultFormData(operationStringToVaultFormAction('DEPOSIT'), amount, true, true, true),
```

Comment:
we probably do need them to acknowledge the terms
![image](https://github.com/user-attachments/assets/89c9598b-b02a-4441-8ca4-7b4906e934b0)

Reply:
Hmmm. The Launch Market flow has its own terms though